### ISSUE TITLE
RCAT 3.0.1? and Anthro 0.0.1- Some Initial Impressions

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/riverscapes-tools#706]
Hey @jtgilbert (and @philipbaileynar FYI; and @stephen-bennett, @wally-mac, @lauren-herbine). I had a play with the preliminary first cut [Anthro](https://drive.google.com/file/d/11IvviIhhwY6kFX-voJ2cCdEUrmKhaHrV/view?usp=sharing) and [RCAT](https://drive.google.com/file/d/11Myo-HeFklRxWiJu-0a13RjVf8gcnKUN/view?usp=sharing) runs for the Middle Bear @jtgilbert shared with me. 

[Here is a video](https://www.youtube.com/watch?v=j9Frc2aW4Tw) with my thoughts and feedback.

Note: These have no business logic or symbology yet (@jtgilbert make those tickets and get lab cranking on these please).

**Comment by lauren-herbine** [comment]
@jtgilbert Does anthro create it's own IGOs..? In the figure below, black dots are IGOs from VBET, and pink dots are from Anthro.
![image](https://user-images.githubusercontent.com/73319684/220769081-b70c2461-3c72-42d4-a796-cd417bcdfe77.png)

Kind of related- how are we going to be able to link Anthro IGOs with all of this awesome data to IGOs of VBET or GNAT? I see there is an IGOID in Anthro, but there isn't anything like that in VBET or GNAT...

**Comment by jtgilbert** [comment]
@lauren-herbine No, it imports them, but there's so many versions of Middle Bear VBET in the warehouse right now that you probably just have a different version than I ran Anthro with. That's a good point about the ID numbers tho, I'll look into that.

**Comment by lauren-herbine** [comment]
@jtgilbert ok, that makes sense! Yesterday I was using the Feb 16th version, but I see today that there is a Feb 22nd one, so I'll check that out

**Comment by philipbaileynar** [comment]
@jtgilbert still relevant? Close if not...
